crates/but/tests/but/journey.rs

```diff
@@ -106,7 +106,10 @@ Caused by:
         .failure()
         .stdout_eq(str![])
         .stderr_eq(str![[r#"
-Error: workspace at refs/heads/main is missing a base
+Error: errors.projects.default_target.not_found
+
+Caused by:
+    there is no default target
```

Comment:
You probably mentioned it, but this feels like a regression.
However, it was also coincidentally 'good' before, and I think this can be ignored until single-branch mode is properly supported.
Now the ID map seems to trigger another related error early, somewhere in old code probably.